solution/src/main/openapi/solution.yaml

```diff
@@ -823,64 +1139,66 @@ components:
           $ref: "#/components/schemas/RunTemplateStepSource"
         parameterGroups:
           type: array
-          description: the ordered list of parameters groups for the Run Template
+          description: The ordered list of parameters groups for the Run Template
           items:
             type: string
-            description: a Run Template Group Parameter id
+            description: A Run Template Group Parameter id
         stackSteps:
           type: boolean
-          description: whether or not to stack adjacent scenario run steps in one container run which will chain steps
+          description: Whether or not to stack adjacent scenario run steps
         gitRepositoryUrl:
           type: string
-          description: an optional URL to the git repository
+          description: An optional URL to the git repository
         gitBranchName:
           type: string
-          description: an optional git branch name
+          description: An optional git branch name
         runTemplateSourceDir:
           type: string
-          description: an optional directory where to find the run template source
+          description: An optional directory where to find the run template source
         orchestratorType:
           $ref: "#/components/schemas/RunTemplateOrchestrator"
         executionTimeout:
           type: integer
-          description: an optional duration in seconds in which a workflow is allowed to run
+          description: An optional duration in seconds in which a workflow is allowed to run
         deleteHistoricalData:
           $ref: "#/components/schemas/DeleteHistoricalData"
       required:
         - id
-#         - name
+
     DeleteHistoricalData:
```

Comment:
Do we still need this in the API? If I recall correctly this was there to cleanup ADX databases and I'm not sure if we still support this with the new internal RDS. If not this should be removed (here or in a subsequent PR) as well as the endpoint using it cleaned. There also seems to be unused related code left in the implementation (see classes `DeleteHistoricalDataOrganization` and `DeleteHistoricalDataWorkspace`, methods `deleteHistoricalDataWorkspace` and `sendDeleteHistoricalDataWorkspaceEvent`, and maybe more).